android/quest/src/main/assets/configurations/app/sync_config.json

```diff
@@ -61,32 +61,28 @@
   },
   {
     "resourceType": "SearchParameter",
-    "name": "publisher",
-    "code": "publisher",
+    "name": "_id",
+    "code": "_id",
     "base": [
       "StructureMap"
-    ],
-    "type": "token",
-    "expression": ""
+    ]
   },
   {
     "resourceType": "SearchParameter",
-    "name": "publisher",
-    "code": "publisher",
+    "name": "_id",
+    "code": "_id",
     "base": [
       "Library"
     ],
     "type": "token",
-    "expression": ""
+    "expression": "1753,46841,46831,46823"
```

Comment:
what are these for?

Reply:
These are Library ids to be loaded. This ensures that during sync it does not load libraries from server and loads only required by app. This sync config would come from server for this ticket #594

Reply:
cool sounds good, I think the issues created for this story will address this, https://github.com/opensrp/fhircore/issues/812